client/src/pages/myProfile.jsx

```diff
@@ -54,22 +55,20 @@ class Profile extends Component{
         <Col md={10}>
         <h2 className="text-dark text-center"> &nbsp; Hi {user}! </h2>
         
-        <Row></Row>
-        <Tabs style={{ marginTop: '2rem'}}
+        <Tabs className={'tutorial2-step-1'} style={{ marginTop: '2rem'}}
               activeKey={isPortfolioSelected ? 'Portfolio': 'History'}
               onSelect={(k) => this.setState({ isPortfolioSelected: k === 'Portfolio' })}
             >
         <Tab eventKey="Portfolio" title="Your Portfolio">
         <Row md={12}>
         <Col md={1}></Col>
         <Col md={10}>
-        {/*<h2 className="text-dark"> &nbsp; {user}'s Portfolio </h2>*/}
         <Portfolio items={items} titleLess={true} light isLoggedIn={!!user} enableClick={false}/>
         </Col></Row>
         <Row md={12}>
         <Col md={1}></Col>
         <Col md={10}>
-            <Tabs
+            <Tabs className={'tutorial2-step-2'}
```

Comment:
This seems like it doesn't have a react-joyride tutorial. Is this intentional or a typo?

Reply:
hey! yes I forgot to remove those. This is from a previous version where the tutorial wasn't interactive.